chia/server/ws_connection.py

```diff
@@ -249,11 +255,40 @@ async def perform_handshake(
             inbound_handshake = Handshake.from_bytes(message.data)
             if inbound_handshake.network_id != network_id:
                 raise ProtocolError(Err.INCOMPATIBLE_NETWORK_ID)
+
+            remote_node_type = NodeType(inbound_handshake.node_type)
```

Comment:
I think this will raise an exception if it's not in the NodeType enum - so I'm not sure the other code will execute

Reply:
I see. If that's the case, I could remove the check and simplify